lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1842,6 +1898,12 @@ ParseResult FIRStmtParser::parsePostFixIntSubscript(Value &result) {
   locationProcessor.setLoc(loc);
   OpBuilder::InsertionGuard guard(builder);
   builder.setInsertionPointAfterValue(result);
```

Comment:
Since you are not adding `bits`operations to the subaccess cache, you should:
1. Don't bother looking up the bits op in the cache
1. Don't create the `bits` op right at the defining value (don't change the builder's insert location)

Or, consider adding them to the cache.

Reply:
I've updated it to the new structure as you recommended above (it is nicer this way, thanks!), so the cache isn't used anymore. The cache could be used without much modification for single-bit accesses, or it could be used for range accesses if it is extended to support that. Do you think doing one or both of those is worth it?

Reply:
I wouldn't bother adding them to the cache.  Adding the cache was super helpful to reduce the amount of IR we generated because of the massive number of subindexes and subfields that show up in the FIRRTL textual format.  I'm guessing that it will be a lot more sparse for bits operations.  One of the first passes we run is CSE which will help clean them up as well.